consul/consul.go

```diff
@@ -101,7 +107,8 @@ func (c *Consul) Register(service *consulapi.AgentServiceRegistration) error {
 	return err
 }
 
-func (c *Consul) register(service *consulapi.AgentServiceRegistration) error {
+func (c *Consul) register(task *tasks.Task, app *apps.App) error {
+	service := c.marathonTaskToConsulService(task, app.HealthChecks, app.Labels)
```

Comment:
I think this should be in `Register` and this function tak care only for comunicating with consul